gradle.properties

```diff
@@ -0,0 +1,5 @@
+org.gradle.daemon=true
+org.gradle.caching=true
+org.gradle.parallel=true
```

Comment:
Since parallel is incubating, we let people opt-in to it still. If we want to enable it on Travis we can include it in the .travis.yml. Earlier it wasn't enabled because it was incubating and we weren't seeing much benefit. However, we've been using parallel without issue for quite a while, are now using the sudo: false infrastructure, and do different things in the build. So if it is helpful now, that's fine.